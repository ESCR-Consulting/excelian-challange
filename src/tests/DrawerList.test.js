import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import DrawerList from '../components/DrawerList';

configure({ adapter: new Adapter() });

describe('DrawerList', () => {
    it('should render correctly', () => {
        const output = shallow(
            <DrawerList />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
