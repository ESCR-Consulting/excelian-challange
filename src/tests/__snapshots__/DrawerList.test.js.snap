// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrawerList should render correctly 1`] = `
<withStyles(List)>
  <withStyles(ListItem)
    button={true}
  >
    <withStyles(ListItemIcon)>
      <pure(AccountCircle) />
    </withStyles(ListItemIcon)>
    <withStyles(ListItemText)
      primary="All Users"
    />
  </withStyles(ListItem)>
  <withStyles(ListItem)
    button={true}
  >
    <withStyles(ListItemIcon)>
      <pure(Star) />
    </withStyles(ListItemIcon)>
    <withStyles(ListItemText)
      primary="Favourites"
    />
  </withStyles(ListItem)>
  <withStyles(ListItem)
    button={true}
  >
    <withStyles(ListItemIcon)>
      <pure(VerifiedUser) />
    </withStyles(ListItemIcon)>
    <withStyles(ListItemText)
      primary="Administrators"
    />
  </withStyles(ListItem)>
  <withStyles(ListItem)
    button={true}
  >
    <withStyles(ListItemIcon)>
      <pure(People) />
    </withStyles(ListItemIcon)>
    <withStyles(ListItemText)
      primary="Non-Admins"
    />
  </withStyles(ListItem)>
  <withStyles(ListItem)
    button={true}
  >
    <withStyles(ListItemIcon)>
      <pure(Inbox) />
    </withStyles(ListItemIcon)>
    <withStyles(ListItemText)
      primary="Archived"
    />
  </withStyles(ListItem)>
</withStyles(List)>
`;
